site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+{{< figure src="binding.png"
+    title="Automatically bind your controls easily." >}}
+
+### Disable HIDAPI drivers due to compatibility problems with certain controllers ([#5123](https://github.com/citra-emu/citra/pull/5123), [#5179](https://github.com/citra-emu/citra/pull/5179)) by [vitor-k](https://github.com/vitor-k)
+
+The change of SDL2 from 2.0.8 to 2.0.10 broke some controllers that reported themselves as a Switch or Xbox One controller, such as DS4 controllers. This PR disables the HIDAPI drivers to allow for those affected controllers to begin to work again. HIDAPI drivers were enabled again for SDL2.0.12 and up in a [later PR](https://github.com/citra-emu/citra/pull/5179).
+
+### Implement official GameCube adapter support ([#5735](https://github.com/citra-emu/citra/pull/5735)) by [epicboy](https://github.com/ameerj)
+
+Ported over from yuzu, with code and feature improvements thanks to [german77](https://github.com/german77) and [lioncash](https://github.com/lioncash), this adds a whole new input device to Citra for you to use! Using the official GameCube adapter from Nintendo, you can now use the classic GameCube controllers to play your favorite games on Citra!
+
+Instructions on how to set this up can be found on the [yuzu FAQ](https://yuzu-emu.org/wiki/faq/#how-do-i-use-my-gamecube-controller-adapter).
+
+### Add support for SDL controller accelerometer/gyro events ([#5851](https://github.com/citra-emu/citra/pull/5851)) by [flibitijibibo](https://github.com/flibitijibibo)
+
+Following the release of SDL 2.0.14 came support for motion controls, something which had been anticipated for many years in SDL. In Citra, this can be used as an alternative to mouse controls or CemuhookUDP. This is a big benefit to many users, as setting up Cemuhook can be tedious and outright confusing at times. 
+Setting up SDL motion controls is as easy as pressing a button! (No, seriously. That is all you have to do.)
+
+{{< figure src="motion.png"
+    title="Motion controls at the press of a button!" >}}
+
+# Android
+
+If you missed the last announcement, this is a follow up to our previous [Citra Android Update](https://citra-emu.org/entry/citra-android-update/)!
+
+We are still updating Citra Android, but all builds are now hosted on our [Nightly repository](https://github.com/citra-emu/citra-nightly/releases) instead of the Google Play Store at the moment. 
+To use all of these brand new features for Android, make sure you have downloaded and installed the latest .apk from our Nightly repository. We will be updating Citra Android on the Google Play Store again some time in the future.
```

Comment:
it could be worth mentioning here, why we are doing this. (Googles new storage permissions system that requires a large rewrite of our android file handling).
And maybe mention that we are actively working on getting updates back on the Play Store.